src/content/1.7/development/webservice/tutorials/2-testing-access.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 2 - Testing access to the Webservice
+menuTitle: 2 - Testing access
+weight: 2
+---
+
+# Testing access to the Webservice
+
+## Accessing /api/
+
+Now that your access key is generated you can test your store's webservice, its endpoint is located in the `/api/` folder at the root of your installation of Prestashop.
```

Comment:
There is no `/api/` folder :thinking:

Reply:
Oh you mean the `/api/` URI :sweat_smile:

Reply:
Yep I'm gonna change that